java-extensions/hudi-reader/pom.xml

```diff
@@ -16,7 +16,7 @@
         <maven.compiler.target>8</maven.compiler.target>
         <java-extensions.home>${basedir}/../</java-extensions.home>
         <slf4j.version>1.7.32</slf4j.version>
-        <presto.hadoop.version>2.7.4-9</presto.hadoop.version>
+        <presto.hadoop.version>2.7.4-11</presto.hadoop.version>
```

Comment:
To resolve external `core-site.xml` can not be loaded. 

see this commit: https://github.com/prestodb/presto-hadoop-apache2/commit/70728f4d47d8bc63b26d867bdd7c20a9a8fa4054